README.md

```diff
@@ -48,7 +49,11 @@ It is an ordered list of roles and/or recipes that are run in the exact order de
 
 `validation_key_format`: Specifies the format in which `validation_key` is set in the `privateconfig.config` file. Supported values are `plaintext` and `base64encoded`. Default value is `plaintext`.
 
-`bootstrap_version`: Set the version of `chef-client` that needs to get installed on the VM. This option is supported only for linux extension.
+`bootstrap_version`: Set the version of `chef-client` that needs to get installed on the VM.
+
+`daemon`: Supported only for windows node. Configures the chef-client service for unattended execution. Supported values are `auto` and `service`. Default is `service`.
+`auto` uses the default behavior. Currently prevents the chef-client service from being configured as a service.
```

Comment:
It is a little confusing that 'service' is the default, but auto uses the default behavior which is to do nothing? Those seem contrary, we should word them so they aren't.